src/components/drawer/ComponentDetailPanel.vue

```diff
@@ -157,7 +164,7 @@ let viewSwitchSubscription;
 /**
  * Update local component data and emit DrawEvent & RenderEvent events.
  */
-async function save() {
+async function submit() {
```

Comment:
at the end of this function, please set `forceSave` to false

Reply:
done